### PR TITLE
Feat/client info header

### PR DESCRIPTION
**Pull request by awalias**
for https://github.com/supabase/infrastructure/issues/1044

### CODE REVIEW

**Comment by awalias** [comment]
oh hold on

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@supabase/supabase-js",
-  "version": "1.0.0",
+  "version": "0.0.0",
```
Comment (awalias) [inline package.json:3]: @inian I think this is safe to revert back to 0.0.0? It's not used anywhere, right?

Comment (soedirgo) [reply package.json:3]: Yeah semantic-release will update this automatically so this is unused.

Comment (awalias) [reply package.json:3]: we might want to have semantic-release also check in this change

**Pull request merged**